lib/models/account.rb

```diff
@@ -1,12 +1,29 @@
 class Account < Forgery
-
-  @local_dictionaries = File.absolute_path("lib")
-  @row
+  attr_reader :name, :uid, :parent_id, :root_id, :time_zone, :sis_id, :workflow
 
   def to_s
-    string = ""
+    string = "#{@name}, #{@uid}, #{@parent_id}, #{@root_id}, #{@time_zone}, #{@sis_id}, #{@workflow}"
+  end
+
+  def initialize(name, uid, parent, root, time_zone, sis, workflow)
```

Comment:
I'm just wondering if all these arguments should be required as not all of them are necessary for creating an account. Not specifying a parent will default to root. Not specifying a time zone will default to match the root and sis IDs aren't required (though maybe they will be for our needs. But I think we could still let the random method initialize accounts with all these arguments but leave things more flexible for the future by allowing an options hash. No?

Reply:
I was fighting with the random stuff early on, so this is the way I was able to get the random init working, but I may have worked out the problems that were causing it. I can mess around with it if you don't like it.

Reply:
I think you may be trying to build in flexibility we won't need.

Reply:
Changes made